codecov.yml

```diff
@@ -0,0 +1,2 @@
+ignore:
```

Comment:
actually I just remembered we can use 

smth like `coverageExcludedPackages := "com\\.project\\.core\\.exclude.*"`

Reply:
`coverageEnabled := false,` does the job.